components/Inputs/RadioButton.vue

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE for license details.
+ */
+
+<template>
+    <div :class="['radio', radioStateClasses]">
```

Comment:
maybe assign `radio` class to `radioStateClasses`. Joining of clesses in one place.